response_operations_ui/controllers/notify_controller.py

```diff
@@ -39,15 +37,17 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
             notification.update({"reference": reference})
 
         url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
+        status_number = response.status_code
+
+        if status_number == 201:
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json().get("id"))
         else:
             ref = reference if reference else 'reference_unknown'
-            raise NotifyError("There was a problem sending a notification to Notify-Gateway to GOV.UK Notify",
-                              reference=ref)
+            logger.error(f'STATUS CODE: {status_number}')
```

Comment:
There's no need to do this kind of logging.  We use structlog for structured logging.  This means that as well as a log message you can attach any number of bits of metadata to it.  For examples, look at pretty much any of the logging in the repo.
This should be something like 
```python
logger.error("There was a problem sending a notification to the notify-gateway", status_code=status_code, error_text=response.text, url=response.url)
```